components/org.wso2.carbon.identity.application.authenticator.fido2.endpoint/src/gen/java/org/wso2/carbon/identity/application/authenticator/fido2/endpoint/ApiException.java

```diff
@@ -0,0 +1,9 @@
+package org.wso2.carbon.identity.application.authenticator.fido2.endpoint;
+//comment
+public class ApiException extends Exception{
```

Comment:
Format the code, missing space  "Exception{". Check code formatting in all the classes

Reply:
This is an auto-generated class.